Code/GraphMol/SubstructLibrary/Wrap/SubstructLibraryWrap.cpp

```diff
@@ -833,7 +832,8 @@ struct substructlibrary_wrapper {
 
         .def("Serialize", &SubstructLibrary_Serialize, python::args("self"))
         // enable pickle support
-        .def_pickle(substructlibrary_pickle_suite());
+        .def_pickle(substructlibrary_pickle_suite()
+		    );
```

Comment:
I suspect that this isn't part of the fix.

Reply:
That was one of the monkeys banging the keyboard.

Reply:
> That was one of the monkeys banging the keyboard.

They just can't be trusted!